compiler/src/main/java/com/squareup/anvil/compiler/codegen/CodeGenerationExtension.kt

```diff
@@ -68,14 +68,20 @@ internal class CodeGenerationExtension(
     }
 
     val flushedFiles = codeGenerators
+      .filterIsInstance<FlushingCodeGenerator>()
       .flatMap { codeGenerator -> codeGenerator.flush(codeGenDir, module) }
       .parse(psiManager)
 
     codeGenerators
       .filterIsInstance<PrivateCodeGenerator>()
       .forEach { codeGenerator ->
         codeGenerator.generateCode(codeGenDir, module, flushedFiles)
-        codeGenerator.flush(codeGenDir, module)
+      }
+
+    codeGenerators
+      .filterIsInstance<FlushingCodeGenerator>()
+      .forEach {
+        it.flush(codeGenDir, module)
```

Comment:
super nit: explicitly use `codeGenerator ->` as lambda parameter for consistency with above.

Reply:
Fair